docs/content/tutorial/7-align-range.md

```diff
@@ -10,10 +10,15 @@ Centered alignment. The effect of this parameter depends on the orientation of
 the chart. For example, on a column chart, elements will be vertically centered, 
 whereas on a bar chart, horizontally.
 
+Note: Be careful, in this first example that the bars no longer make sense
```

Comment:
Note: In the first example, the y-axis labels are hidden because they don't properly represent the values shown on the column chart anymore, as the chart elements float off the x-axis.